quickannotator/api/v1/annotation/routes.py

```diff
@@ -129,4 +139,78 @@ def post(self, args):
             resp['centroid'] = json.dumps(mapping(union.centroid))   
             resp['area'] = union.area
 
-        return resp, 200
\ No newline at end of file
+        return resp, 200
+    
+# @bp.route('/download/<int:image_id>/<int:annotation_class_id>/<str:format>')
+# class DownloadAnnotations(MethodView):
+#     def get(self, image_id, annotation_class_id, format):
+#         """   download a tar archive corresponding to a single image and annotation class   """
+#         # check if a tar file in the correct format already exists within the directory structure.
+
+@bp.route('/export/local')
+class DownloadAnnotations(MethodView):
+    @bp.arguments(server_models.DownloadAnnsArgsSchema, location='json')
+    def post(self, args):
+        """ Export annotations for multiple images and annotation classes as a TAR file with progress updates """
+
+        image_ids = np.array(args['image_ids'])
+        annotation_class_ids = np.array(args['annotation_class_ids'])
+        annotations_format = args.get('format', 'geojson')
+
+        # Compute the cartesian product of image_ids and annotation_class_ids
+        image_ids_grid, annotation_class_ids_grid = np.meshgrid(image_ids, annotation_class_ids, indexing='ij')
```

Comment:
itertools is probably a bit more natural here? https://docs.python.org/3/library/itertools.html#itertools.product